docs/content/concepts/ops-jobs-graphs/metadata-tags.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: Metadata & Tags | Dagster
+description: Metadata and tags provide two ways to attach information in Dagster
+---
+
+# Metadata & Tags
+
+Metadata and tags provide two different ways for you to attach information to your jobs and the runs launched from those jobs. The main difference between metadata and tags is what "object" the information is attached to. Metadata is attached to JobDefinitions (specified using the @job decorator) and tags are attached to runs that are created by executing a job.
+
+## Metadata
+
+Metadata allows you to attach information to a job. This information can be whatever you want, but possible cases include:
+
+- keeping track of the team responsible for maintaining a job
+- linking to documentation or other resources
```

Comment:
@sryza @dpeng817 what are some other good uses of metadata to suggest?

Reply:
Can link git hash that corresponds to current definition